### ISSUE TITLE
Improve error msg when VT Get Report does not have an entry for

### ISSUE THREAD

**Issue by mnmnc** [TheHive-Project/Cortex-Analyzers#248]
### Request Type
Bug

### Work Environment
| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RedHat
| Cortex Analyzer Name      |VirusTotal
| Cortex Analyzer Version   | 3.0
| Cortex Version            | 2.0.4

### Description
IP added as observable. Type: ip
Added as 1.1.1.1, presented in UI as 1[.]1[.]1[.]1
I think VT analyser is taking literal 1[.]1[.]1[.]1 instead of 1.1.1.1 which causes an error (i've changed the IP) displayed in UI as:  

```
Report for VirusTotal_GetReport_3_0 analysis of Thu, May 17th, 2018 14:27 +02:00 Show Raw Report
1[.]1[.]1[.]1
Missing IP address
```

### Steps to Reproduce
(keep this section only if the issue relates to a bug)
1. add IP as observable to case
1. start VirusTotal_GetReport analyser from observable details page
1. get error

### Possible Solutions
```python
given_ip.replace('[', '').replace(']', '')
```
or something better ;)

**Comment by saadkadhi** [comment]
@mnmnc the problem might be on your side. I've just tested with 1.1.1.1 added as an observable and VT GetReport works like a charm. Also tried directly from Cortex 2.0.4 and everything works perfectly. Make sure you have no extra chars/space at the beginning/end of the IP addr.

<img width="398" alt="screen shot 2018-05-17 at 09 58 59" src="https://user-images.githubusercontent.com/2852427/40182323-260f11d0-59b9-11e8-831f-ec042715f08b.png">

<img width="1226" alt="screen shot 2018-05-17 at 10 00 39" src="https://user-images.githubusercontent.com/2852427/40182388-47ee43b6-59b9-11e8-97e9-ccca54512358.png">

<img width="1404" alt="screen shot 2018-05-17 at 10 00 49" src="https://user-images.githubusercontent.com/2852427/40182400-4f4d1a9c-59b9-11e8-87ac-f6c38401daca.png">

Internally, the IP is stored as-is. It is displayed fanged and when you attempt to export it without unfanging it first, you'll get it obviously fanged.

**Comment by mnmnc** [comment]
Ok. I will check everything on my side.
Here it looks like this at present: 

![Fail](https://raw.githubusercontent.com/mnmnc/img/master/vt.png)

**Comment by mnmnc** [comment]
Ok. I've tested on a bunch of IP addresses.   
So apperently "Missing IP address" is a response in case VirusTotal does not have this IP in a database.  Which in my opinion is pretty misleading response.   
If this can be clarified on a analyzer level ("VT does not have information about this IP") that would be great.   
Currently It looks like a user fault of not providing IP address.

**Comment by saadkadhi** [comment]
Thanks for testing further. Obviously it has 1.1.1.1 in its database that's why I could not reproduce the problem on my end per your initial issue description ;-).

**Comment by jeromeleonard** [comment]
Hi, 

in fact, this is the message that is returned by VirusTotal API when IP is not in the database.

**Comment by 3c7** [comment]
Fixed with 8392531.
